kornia/core/tensor_wrapper.py

```diff
@@ -1,3 +1,4 @@
+# type: ignore
```

Comment:
It needs a specific issue to type this file (I can't figure out how to do this).

Also, needs to remove the `# type: ignore[attr-defined]` from the files which imports the tensor wrapper

Reply:
I added this import type ignore on the files:
- kornia/geometry/plane.py
- kornia/geometry/vector.py

Reply:
@cclauss

Reply:
I would be OK with @johnnv1 proposed approach to land these changes with line 1 as is as long as a new issue is opened to get rid of it.

The way to do this (in a separate PR) is to remove this comment and then run `mypy` (or another typing tool) and see what type errors are raised and then specify `# type: ignore[your-issue-here]` on individual lines.